include/seqan3/core/simd/simd_algorithm.hpp

```diff
@@ -373,6 +383,16 @@ constexpr void transpose(std::array<simd_t, simd_traits<simd_t>::length> & matri
 {
     detail::transpose_matrix_sse4(matrix);
 }
+// AVX2 implementation
+template <simd::simd_concept simd_t>
+    requires detail::is_builtin_simd_v<simd_t> &&
+             detail::is_native_builtin_simd_v<simd_t> &&
+             simd_traits<simd_t>::max_length == 32 &&
+             simd_traits<simd_t>::length == 32
```

Comment:
```suggestion
             (simd_traits<simd_t>::max_length == 32) &&
             (simd_traits<simd_t>::length == 32)
```

I saw the conflicts and we need parenthesis here